app/scripts/modules/core/cluster/all.html

```diff
@@ -45,10 +45,14 @@
   </div>
   <filter-tags tags="tags" tag-cleared="allClusters.updateClusterGroups()" clear-filters="allClusters.clearFilters()"></filter-tags>
 </div>
-<div class="content"
-     data-scroll-id="clusters-content"
-     sticky-headers
-     waypoint-container
-     waypoint-container-key="{{application.name}}"
-     waypoint-offset="100"
-     ng-include="allClusters.groupingsTemplate"></div>
+
+
+<all-clusters-groupings
+  class="content"
+  sticky-headers
+
+  groups="groups"
+  sortFilter="sortFilter"
+  app="application"
+  initialized="allClusters.initialized"
+></all-clusters-groupings>
```

Comment:
We don't have documented styles around spacing/indenting of nodes, but this would look closer to the rest of the codebase (in theory) as:
```html
<all-clusters-groupings class="content"
                        sticky-headers
                        groups="groups"
                        sortFilter="sortFilter"
                        app="application"
                        initialized="allClusters.initialized"></all-clusters-groupings>
``` 
first attribute on same line as tag, other attributes indented and aligned with first attribute are the main styles we generally adhere to; placement of `>` is kind of an open question